src/runtime/pkg/govmm/qemu/qemu.go

```diff
@@ -1936,6 +1941,12 @@ func (vfioDev VFIODevice) QemuParams(config *Config) []string {
 		deviceParams = append(deviceParams, fmt.Sprintf("devno=%s", vfioDev.DevNo))
 	}
 
+	if strings.HasPrefix(vfioDev.SysfsDev, drivers.IommufdDevPath) {
```

Comment:
Same here, this makes readability a little bit worse because `SysfsDev` should start with `/sys` in a common sense.